docs/setup.md

```diff
@@ -190,9 +190,9 @@ Once a custom package has been added to Private Packagist you will benefit from
 
 ### Prevent a package from being used with custom packages
 
-While there is no direct "block package" feature in Packagist, you can effectively prevent a specific package from being used by creating a placeholder package. 
+While there is no direct "block package" feature in Packagist, you can effectively prevent a specific package from being used in your project by creating a placeholder package. 
```

Comment:
instead of saying project we could generalize it and say Composer repository? Or (sub-)organization?
Project (for me) is really just a package with a composer.json + .lock file

Reply:
I'd suggest "your Composer project" then? It's about blocking usage in your codebase, not in the Private Packagist org/repository?

Reply:
oh i see how you look at it. Yeah then leave it as is